src/Components/Pages/RentCalculator/RentCalculator.scss

```
@@ -0,0 +1,249 @@
+@import "../../../mixins.scss";
+@import "../../../colors.scss";
+
+#rent-calculator-page {
+  .headline-section {
+    padding-top: 2.5rem;
+
+    .headline-section__subtitle {
+      width: 85%;
+    }
+  }
+  .content-section {
+    padding-top: 2.25rem;
+    @media (min-width: 773px) {
+      padding: 2.25rem 3rem 0;
+    }
+    @include for-phone-only {
+      padding: 0 1rem;
+    }
+
+    .rent-increase-explanation {
+      width: 90%;
+      @include body_large_desktop;
+      @include for-phone-only {
+        @include body_large_mobile;
+        font-weight: 500;
+        margin: 1rem 0;
+      }
+      .jfcl-link {
+        @include body_large_desktop;
+        svg {
+          display: none; // hide internal link arrow icon
+        }
+      }
+      .mobile-breakpoint {
+        @include for-tablet-portrait-up {
+          display: none;
+        }
+      }
+    }
+
+    .content-box {
+      margin: 1.5rem 0;
+      @include for-phone-only {
+        margin: 1rem 0;
+      }
+    }
+  }
+
+  .rent-calculator-callout-box {
+    display: flex;
+    flex-direction: column;
+    gap: 1.5rem; // 24px
+    padding: 2rem; // 32px
+    background-color: $OFF_WHITE_200;
+    border-radius: 4px;
+    margin-bottom: 1rem;
+    @include for-phone-only() {
+      flex-direction: column;
+      padding: 1.5rem; // 24px
+    }
+
+    .callout-box__header {
+      @include body_large_desktop;
+      font-weight: 600;
+    }
+  }
+
+  .rent-input-container {
+    align-items: flex-end;
+    display: grid;
+    grid-template-columns: 2fr 1fr;
+    @include for-phone-only {
+      grid-template-columns: 1fr;
+    }
+
+    @include for-phone-only() {
+      width: -webkit-fill-available;
+      .phone-number-submit__desktop {
+        display: none;
+      }
+    }
+
+    label {
+      @include body_standard_desktop;
+      font-weight: 500;
+    }
+    .jfcl-input-header__invalid_text {
+      margin-bottom: 4px;
+    }
+    .jfcl-text-input {
+      margin: unset;
+      margin-right: 1rem;
+      & :focus-visible {
+        outline-color: blue;
+      }
+      @include for-phone-only {
+        width: 100%;
+      }
+    }
```

On the survey page we hide these number increment buttons from the input. In case we want to do the same here, these are the styles:

```suggestion
    }
    #rent-input {
      &::-webkit-inner-spin-button,
      &::-webkit-outer-spin-button {
        appearance: none;
        margin: 0;
      }
    }
```